templates/settings.php

```diff
@@ -89,6 +90,24 @@
                     </select>
                     <input type='submit' name='change_layout' value='Change' />
                 </form>
+                <?php if($player->layout == "new_geisha"): ?>
+                <label>Avatar Style</label>
+                <form action='<?=$system->router->getUrl('settings')?>' method='post'>
+                    <select name='avatar_style'>";
+                        <option value='round' <?=($avatar_style == "round" ? "selected='selected'" : "")?>>Round</option>
+                        <option value='four-point' <?=($avatar_style == "four-point" ? "selected='selected'" : "")?>>Four-Point</option>
```

Comment:
```suggestion
                        <option value='four-point' <?=($avatar_style == "four-point" ? "selected='selected'" : "")?>>Square</option>
```
I think people will know this more as "square" although we could also totally include a preview (especially when we add in more)